src/test/java/cms/gov/madie/measure/utils/MeasureUtilTest.java

```diff
@@ -606,6 +607,41 @@ public void testIsCqlLibraryNameChangedReturnsTrueForChange() {
     assertThat(output, is(true));
   }
 
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForNullSupplementalData() {
+    final Measure original = Measure.builder().supplementalData(Collections.emptyList()).build();
+    final Measure changed = original.toBuilder().supplementalData(Collections.emptyList()).build();
+    boolean output = measureUtil.isSupplementalDataChanged(changed, original);
+    assertThat(output, is(false));
+  }
+
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForOriginalNonNullSupplementalData() {
+    SupplementalData supplementalData1 =
+        SupplementalData.builder()
+            .definition("THIS_DEFINITION")
+            .description("Just a dumb definition")
+            .build();
+    final Measure original = Measure.builder().supplementalData(List.of(supplementalData1)).build();
+    final Measure changed = Measure.builder().supplementalData(Collections.emptyList()).build();
+    boolean output = measureUtil.isSupplementalDataChanged(changed, original);
+    assertThat(output, is(true));
+  }
+
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForChangedNonNullSupplementalData() {
```

Comment:
Please rename method. Test returns true for new SDE.
```suggestion
  public void testIsSupplementalDataChanged_ReturnsTrueForNewNonNullSupplementalData() {
```